subprojects/language-native/src/test/groovy/org/gradle/language/c/tasks/CCompileTest.groovy

```diff
@@ -61,8 +61,8 @@ class CCompileTest extends AbstractProjectBuilderSpec {
         platform.getName() >> "testPlatform"
         platform.getArchitecture() >> Mock(ArchitectureInternal) { getName() >> "arch" }
         platform.getOperatingSystem() >> Mock(OperatingSystemInternal) { getName() >> "os" }
-        1 * toolChain.select(platform) >> platformToolChain
-        1 * platformToolChain.newCompiler({CCompileSpec.class.isAssignableFrom(it)}) >> cCompiler
+        _ * toolChain.select(platform) >> platformToolChain
+        _ * platformToolChain.newCompiler({CCompileSpec.class.isAssignableFrom(it)}) >> cCompiler
```

Comment:
Does this need to be dynamic? Can we lock it down to a specific number perhaps?

Reply:
I locked down the number to 7.